lib/scoreTest.js

```diff
@@ -85,16 +113,39 @@ function scoreOneExpectation(expected, apiResults, context) {
 }
 
 /**
- * Score an entire test by combining the score for each expectation, and the
- * score for any unexpected properties.
+ * Score one (out of potentially many) coordinate expectation
+ * independently of properties
+ */
+function scoreOneCoordinateExpectation(expected_coords, apiResults, context) {
+  return apiResults.map(function(result) {
+    var score = scoreOneCoordinateResult(expected_coords, result, apiResults, context);
+    return score;
+  }).sort(sortScores)[0];
+}
+
+/**
+ * Score an entire test by combining the score for each expectation,
+ * and the score for any unexpected properties.
  */
 function scoreTest(testCase, apiResults, context) {
   var scores = [];
+  var coordinates = testCase.expected.coordinates || [];
```

Comment:
One of the unit tests is failing because of this line, looks like all you have to do is move it into the `if (testCase.expected)` block.